.github/azure-autodoc.yml

```diff
@@ -25,6 +25,14 @@ steps:
       jdkVersionOption: "1.11"
       goals: install
       options: $(MVN) -T 1C -DskipTests -pl !executable
+  - task: Bash@3
```

Comment:
Cool

Reply:
Das hat bei mir davor zu komischen Fehlern geführt deswegen so in 2 Schritten